packages/react-components/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -122,7 +135,10 @@ const useRootStyles = makeStyles({
   underline: {
     backgroundColor: tokens.colorTransparentBackground,
     ...shorthands.borderRadius(0), // corners look strange if rounded
-    ...shorthands.borderBottom('1px', 'solid', tokens.colorNeutralStrokeAccessible),
+    // border is specified in rootBaseStyles, but we only want a bottom border here
+    borderTopStyle: 'none',
+    borderRightStyle: 'none',
+    borderLeftStyle: 'none',
```

Comment:
Should you maybe make the color 'transparent' instead of the style 'none' so that the border is still visible in HCM (I don't know what the requirements are for the underlined variant in HCM, so if the expectation is still only for the underline to show feel free to resolve this comment).

Reply:
My goal is to not make any visual changes to the styles. It previously only had a bottom border for `underline`, and I don't want to change that in this PR.